### PR TITLE
Devise for signup login

### PR DESCRIPTION
**Pull request by ghost**
### What we did:

- Add devise gem
- Add login, signup, sign out to navbar
- Generate user model

### CODE REVIEW

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+
```
Comment (alghazu) [inline spec/factories/users.rb]: Create factory bot for users

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (alghazu) [inline spec/models/user_spec.rb]: Add rspec test, or delete this

**Review by alghazu — CHANGES_REQUESTED** [review]

File: spec/models/user_spec.rb (hunk 2 above)
Comment (ghost) [reply spec/models/user_spec.rb]: Add rspec test for email and password

**Review by alghazu — APPROVED** [review]

**Comment by alghazu** [comment]
Looks good to me

File: spec/models/site.css.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+.navbar {
+    overflow: hidden;
+    background-color: white;
+    text-shadow: 4px 4px 5px white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
```
Comment (tochman) [inline spec/models/site.css.scss]: Why is this css here?

**Review by tochman — CHANGES_REQUESTED** [review]
In what way is this acceptance criteria fullfilled?: Restrict access to articles#show unless there is a `current_user`

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { create(:user) }
```
Comment (tochman) [inline spec/models/user_spec.rb]: Why are you setting this variable/creating this object?

File: spec/models/site.css.scss (hunk 3 above)
Comment (GeorgeJLewis) [reply spec/models/site.css.scss]: removed, it should not have been there.

**Comment by tochman** [comment]
Sorry, not mergeable. Please redo this feature.

**Pull request closed without merging**